gateway-service/src/main/java/org/zowe/apiml/gateway/config/EurekaEventsRegistry.java

```diff
@@ -25,7 +25,7 @@
 public class EurekaEventsRegistry {
 
     private final EurekaClient eurekaClient;
-    private final List<MetadataProcessor> processors;
+    private final List<EurekaEventListener> processors;
```

Comment:
Are there any left? Could this class be removed?

Reply:
not used anymore, class deleted